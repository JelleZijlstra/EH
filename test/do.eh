echo 1

do
    echo 2
    echo 3
end

echo 4

private f x = do
    echo x
    do
        echo(x + x)
    end
    x
end

echo (f 5)
