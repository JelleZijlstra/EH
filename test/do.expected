%%stdout%%
1
2
3
4
5
10
5
%%stderr%%
